ur_robot_driver/CMakeLists.txt

```diff
@@ -88,6 +90,8 @@ add_library(ur_robot_driver
     src/primary/robot_message.cpp
     src/primary/robot_message/version_message.cpp
     src/primary/robot_state/kinematics_info.cpp
+    src/ros/dashboard_client_ros.cpp
+    src/ros/hardware_interface.cpp
```

Comment:
That was actually on purpose, so that the ur_robot_driver lib can be used without ROS. Since the plugin needs a library I see the problem there.

To add some perspective: I plan to move out the Non-ROS related stuff into a standalone plain cmake package. This will one day also make the ROS2 migration easier to do.

Therefore, I suggest to create another lib containing the ROS parts, namely those two source files and then link the driver_node to that as well.

Reply:
This makes sense. I included your comments in 344e6d4cd13a27ee3cc21b2b18e81fa03d369b5e.